src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -78,10 +82,99 @@ public AuditUtility(
             }
         }
 
+        private void CheckDownloadDependenciesVulnerabilities(IReadOnlyList<IReadOnlyDictionary<string, IReadOnlyList<PackageVulnerabilityInfo>>> knownVulnerabilities)
```

Comment:
There's a mixture of naming between `DownloadDependencies` and `PackageDownloads`.  Since in the code its already called `DownloadDependencies`, we should probably just stick with that?  The MSBuild items of course are `<PackageDownload />` 🤦